src/clj/harja/kyselyt/tehtavaryhmat.sql

```diff
@@ -3,11 +3,34 @@ SELECT tro.id, tro.otsikko
   FROM tehtavaryhmaotsikko tro
  ORDER BY tro.otsikko ASC;
 
+-- name: hae-tehtavaryhma
+SELECT tro.id        AS tehtavaryhmaotsikko_id,
+       tro.otsikko,
+       tr.id         AS tehtavaryhma_id,
+       tr.nimi,
+       tr.voimassaolo_alkuvuosi,
+       tr.voimassaolo_loppuvuosi,
+       tr.jarjestys,
+       tr.nakyva,
+       tr.versio,
+       tr.yksiloiva_tunniste,
+       JSONB_AGG(ROW_TO_JSON(ROW (t.id, t.nimi, t.yksikko, t.jarjestys, t.api_seuranta, t.suoritettavatehtava,
```

Comment:
Käytetäänkö näitä ollenkaan? Omaan silmään koodista ei pistä kohtaa, missä näillä tehtäisiin mitään.